frontend/src/components/FileTableRowComponent.vue

```diff
@@ -49,12 +44,12 @@
       >
         <font-awesome-icon class="text-danger fs-5" :icon="['fas', 'xmark']" />
       </span>
-      <span v-else>
-        <font-awesome-icon class="text-primary fs-5" :icon="['fas', 'check']" />
-      </span>
+      <div v-else class="spinner-border text-secondary" role="status">
+        <span class="visually-hidden">Loading...</span>
```

Comment:
We've lost the progress indicator btw ; this is problematic. Uploading 100MB on a slow line is **very** long and user needs to see progress. Once it's _uploaded_ (but not in S3), we can switch the number for _Processing_